File: backend/voting-app/src/main/java/com/tu/votingapp/dto/general/survey/SurveyResponseDTO.java
package com.tu.votingapp.dto.general.survey;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SurveyResponseDTO {
    private Long id;
    private Long userId;
    private Long surveyId;
    private Long questionId;
    private Long optionId;
    private LocalDateTime respondedAt;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/survey/SurveyResponseDTO.kt
package com.example.android.dto.response.survey

import kotlinx.serialization.Serializable

@Serializable
data class SurveyResponseDTO(
    val id: Long,
    val userId: Long,
    val surveyId: Long,
    val questionId: Long,
    val optionId: Long,
    val respondedAt: String // ISO 8601 date-time string (e.g., "2025-04-19T14:30:00")
)